scripts/deploy.ts

```diff
@@ -117,18 +118,15 @@ async function main() {
   // ForwarderV4 and ForwarderFactoryV4.
   // If we have to deploy contracts for the older coins like eth, avax, polygon, we need to deploy Forwarder and ForwarderFactory
   const Forwarder = await ethers.getContractFactory(forwarderContractName);
-  const forwarder = await Forwarder.deploy(gasParams);
+  const forwarder = await Forwarder.deploy();
```

Comment:
maybe we can add eip1559GasParams here and in case of forwarder factory as well

Reply:
seems like the contracts got deployed without this change on local, so not needed